iwa-console/PublicAppUsingIntegratedWindowsAuthentication.cs

```diff
@@ -43,11 +43,11 @@ public class PublicAppUsingIntegratedWindowsAuthentication
         /// <remarks>
         /// For more information see https://aka.ms/msal-net-iwa
         /// </remarks>
-        public PublicAppUsingIntegratedWindowsAuthentication(PublicClientApplication app)
+        public PublicAppUsingIntegratedWindowsAuthentication(IPublicClientApplication app)
         {
             App = app;
         }
-        protected PublicClientApplication App { get; private set; }
+        protected IPublicClientApplication App { get; private set; }
```

Comment:
This is the big (breaking) change: Now that the build returns an IPublicClientApplication, everywhere we had PublicClientApplication in the code, we change it to IPublicClientApplication

Reply:
I agree it's a breaking change, but why is it big? Just add an "I" :).

Reply:
this takes time :)